setup-tdx-common

```diff
@@ -40,7 +39,7 @@ EOF
 
   cat <<EOF | tee /etc/apt/apt.conf.d/99unattended-upgrades-kobuk-${ppa_id}
 Unattended-Upgrade::Allowed-Origins {
-  "${distro_id}:${distro_codename}";
+  "${distro_id}:${UBUNTU_CODENAME}";
```

Comment:
I'm wondering if we should add `UBUNTU_CODENAME=$(lsb_release -cs)` to this script? I know it gets sourced by other scripts that are responsible for setting this variable, but it feels like that could be easily overlooked or prone to errors in the future. Or maybe just include a quick check to futureproof it with something like:

```bash
if [[ -z "${UBUNTU_CODENAME}" ]]; then
  >&2 echo "Please set UBUNTU_CODENAME prior to sourcing setup-tdx-common." 
  exit 1
fi
```

Reply:
You are right, let me propose something different

Reply:
@frenchwr i would like to propose that 0d3c308a7b3ac35f1f30bafe2846644399827005 , let me what you thiink